Jellyfin.Server/StartupOptions.cs

```diff
@@ -11,6 +11,9 @@ public class StartupOptions : IStartupOptions
         [Option('d', "datadir", Required = false, HelpText = "Path to use for the data folder (database files, etc.).")]
         public string DataDir { get; set; }
 
+        [Option('w', "webdir", Required = false, HelpText = "Path to the Jellyfin web resources.")]
```

Comment:
```suggestion
        [Option('w', "webdir", Required = false, HelpText = "Path to the Jellyfin web UI resources.")]
```
?

Reply:
Yea that works, I'll have to change that in a few places.

Reply:
Done.